Packs/CimTrak-SystemIntegrityAssurance/Integrations/CimTrak/CimTrak.yml

```diff
@@ -0,0 +1,2328 @@
+category: File Integrity Management
+commonfields:
+  id: CimTrak
+  version: -1
+configuration:
+- defaultvalue: https://127.0.0.1
+  display: App Server URL
+  name: url
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: API Key
+  name: apikey
+  required: true
+  type: 4
+- defaultvalue: "false"
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: 127.0.0.1
+  display: Repository  URL
+  name: Repository URL
+  required: true
+  type: 0
+- defaultvalue: "3749"
+  display: Repository Port
+  name: Repository Port
+  required: true
+  type: 0
+- defaultvalue: "1"
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+- display: Long running instance
+  name: longRunning
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ""
+    itemVersion: ""
+    packID: 25043b93-4265-49a0-8858-04aed0ec4edd
+    propagationLabels: []
+    toServerVersion: ""
+description: The CimTrak integration helps you detect unexpected system/device/config
+  modifications and automatically respond/react to threats
+display: CimTrak - System Integrity Assurance
+name: CimTrak
+script:
+  commands:
+  - arguments:
+    - defaultValue: "1"
+      description: Starting number of record to get
+      name: Start
+    - defaultValue: "50"
+      description: Ending number of record to get
+      name: End
+    - description: 'Filter array to limit results IE: [{name: id, operator:>, value:5}]'
+      name: Filter
+    - description: 'Sort array to sort data IE: [{field: id, descending: False}]'
+      name: Sorts
+    description: Returns Events from the event log
+    name: get-events
+    outputs:
```

Comment:
All outputs of commands (integrations and scripts) should have a description in the YML file and corresponding README.md file